ambry-router/src/main/java/com/github/ambry/router/OperationQuotaCharger.java

```diff
@@ -98,7 +98,9 @@ public QuotaAction checkAndCharge(boolean shouldCheckExceedAllowed) {
   @Override
   public QuotaResource getQuotaResource() {
     if (Objects.isNull(quotaChargeCallback)) {
-      LOGGER.warn("quota charge callback is null for operation: {}", operationName);
+      if (!operationName.equals("ReplicateBlobOperation")) {
```

Comment:
nit: can we do`ReplicateBlobOperation.class.getSimpleName()`?

Reply:
Done